lib/src/signed_video_h26x_sign.c

```diff
@@ -451,12 +453,12 @@ add_nalu_to_prepend(signed_video_t *self,
   assert(data_size > 0);
 
   signed_video_nalu_to_prepend_t *nalu_to_prepend =
-      &(self->nalus_to_prepend_list[self->num_nalus_to_prepend]);
+      &(self->nalus_to_prepend_list[self->num_of_completed_seis]);
```

Comment:
This list is not needed anymore. Hence, this function could be removed/replaced as well.